### PR TITLE
#495 - GetPermissionSlice implemented

### PR DESCRIPTION
**Pull request by olenagerasimova**
Implemented `GetPermissionSlice` for #495

### CODE REVIEW

File: src/main/java/com/artipie/api/ArtipieApi.java [hunk 1]
```diff
@@ -62,6 +63,7 @@
  * Artipie API endpoints.
  * @since 0.6
  * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
+ * @checkstyle ClassFanOutComplexityCheck (500 lines)
```
Comment (olegmoz) [inline src/main/java/com/artipie/api/ArtipieApi.java:66]: @olenagerasimova I'd suggest to write a puzzle about reducing this class complexity

**Review by olegmoz — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, all is good except one thing, please see the comment

File: src/main/java/com/artipie/api/ArtipieApi.java (hunk 1 above)
Comment (olenagerasimova) [reply src/main/java/com/artipie/api/ArtipieApi.java:66]: @olegmoz this class already has a puzzle to decompose constructor https://github.com/artipie/artipie/issues/464, I think that's enough considering the fact that it has no methods, ctor only.

Comment (olegmoz) [reply src/main/java/com/artipie/api/ArtipieApi.java:66]: @olenagerasimova ah, I see. That's clearly enough, thanks

**Review by olegmoz — APPROVED** [review]

**Comment by 0crat** [comment]
Job `gh:artipie/artipie#505` is not assigned, can't get performer

**Pull request merged**